src/main/resources/duraspace-checkstyle/checkstyle.xml

```diff
@@ -95,11 +96,8 @@
         <module name="JavadocMethod">
             <!-- All public methods MUST HAVE Javadocs -->
             <property name="scope" value="public"/>
-            <!-- Allow RuntimeExceptions to be undeclared -->
-            <property name="allowUndeclaredRTE" value="true"/>

```

Comment:
Did the upgrade make this and `allowMissingThrowsTags` obsolete?

Reply:
It appears that those properties were removed due to inconsistent results: https://github.com/checkstyle/checkstyle/issues/7329